cvat/apps/engine/cache.py

```diff
@@ -26,41 +27,46 @@
 
 
 class CacheInteraction:
+    @dataclass()
```

Comment:
I'm not sure I understand why you have added CacheItem... In public methods, you return a tuple but inside internal methods, you return CacheItem. What is the point?

Reply:
Nothing special here, just an attempt to unify the data type between all class methods by adding a simple interface instead of using a dictionary.

Reply:
CacheItem -> tuple